feature-examples/python/corpus/read_corpus.py

```diff
@@ -0,0 +1,133 @@
+"""Example of using the Vectara REST API to read the corpus info."""
+
+import dataclasses
+import logging
+
+import requests
+
+from corpus import exceptions
+
+
+@dataclasses.dataclass(frozen=True)
+class Corpus:
+    """Basic Corpus data."""
+
+    corpus_id: int
+    name: str
+    description: str
+    dt_provisioned: str
+    enabled: bool
+
+
+@dataclasses.dataclass(frozen=True)
+class CorpusSize:
```

Comment:
duplicated in compute_corpus_size. Consider extracting dataclass definitions to a common location.